settings.sample.py

```diff
@@ -159,6 +159,10 @@
 PUSHOVER_APP_TOKEN: str = '0123456789abcdefghijklmnopqrst'
 
 
+# Synology Chat 
+SYNOLOGYCHAT_ENABLED: bool = False
+SYNOLOGYCHAT_WEBHOOK_URL: str = 'http://diskstation.local:5000/webapi/entry.cgi?api=SYNO.Chat.External&method=incoming&version=2&token=%22xxx%22'
```

Comment:
Should be `SYNOLOGYCHAT_WEBHOOK_URL: str = 'http://diskstation.local:5000/webapi/entry.cgi'`
Everything behind it is called URL params, which are usually cleaner to build as a dict and handle with dedicated settings – e.g. `SYNOLOGYCHAT_TOKEN: str = 'whatever-it-looks-like`

and then add this in `alerts.py`

```python
params = {
    'api': 'SYNO.Chat.External',
    'method': 'incoming',
    'version': 2
    'token': settings.SYNOLOGYCHAT_TOKEN
}

response = requests.post(settings.SYNOLOGYCHAT_WEBHOOK_URL, params=params, json=payload, verify=False)
```

Reply:
This is correct, but i thought it would be easier to all to just copy complete URL into setting.